pkg/lib/verifier/rulebuilder_intel_tpm20.go

```diff
@@ -203,8 +203,8 @@ func (builder *ruleBuilderIntelTpm20) GetHostUniqueRules() ([]rules.Rule, error)
 	//
 	// Add 'PcrEventLogIntegrity' rules...
 	//
-	tbootInstalled := builder.signedFlavor.Flavor.Meta.Description.TbootInstalled
-	if tbootInstalled != nil && *tbootInstalled {
+	tbootInstalled := builder.signedFlavor.Flavor.Meta.Description[flavormodel.TbootInstalled]
+	if tbootInstalled != nil {
```

Comment:
The rulebuilders are no longer needed and should defer to templates (perhaps we keep the hardcoded vmware rulebuilder).  But generally, this hard coded validation (ex. if tboot installed so check PCR17) should be deferred to the template.  Let me know if you want to sync.

Reply:
The changes related to flavor verification are in pull request : https://github.com/pkumarex/intel-secl/pull/5